### ISSUE TITLE
Localization support

### ISSUE THREAD

**Issue by snovak7** [PiranhaCMS/piranha.core#425]
How is support for multiple languages?

**Comment by tidyui** [comment]
Totally depends on what you mean by multiple languages. Piranha is **not** truly multi-lingual in the sense that a single page can exist in different languages and you can view the same site structure in different translations.

However you can create multiple sites, mapping to different hostnames that can share information and uploaded media. As of `5.3` which will soon be release you will also be able to specify **culture** on site meaning the requests for content in a site will get the **Culture** and **UICulture** set.

**Comment by snovak7** [comment]
Yes, I am looking that I can translate content, so I don't have to have two separate sites. So I could specify with Culture (UICulture) what translation I want, or fallback language in case it doesn't exist, for example.

**Comment by snovak7** [comment]
No plans for that in the future as an enhancement?

**Comment by tidyui** [comment]
I wouldn't rule it out completely, but to be honest it hasn't been implemented in any version of Piranha since 2011, primarily because the focus of the CMS is to handle small to medium-sized websites. Projects that need true multi-lingual support are often bigger and developers often then go for a bigger CMS, like for example Umbraco.

On the other hand the CMS options are limited for .NET Core which of course leave the market open for implementing good support for it.

Regards

**Comment by steelspace** [comment]
> Totally depends on what you mean by multiple languages. Piranha is **not** truly multi-lingual in the sense that a single page can exist in different languages and you can view the same site structure in different translations.
> 
> However you can create multiple sites, mapping to different hostnames that can share information and uploaded media. As of `5.3` which will soon be release you will also be able to specify **culture** on site meaning the requests for content in a site will get the **Culture** and **UICulture** set.

Can I use just /{lng}/ instead of a different hostname? (v 5.3)

**Comment by tidyui** [comment]
Yes, it's totally fine, if you set it up like this

* Default site `mydomain.com`
* Other site `mydomain.com/lng`

Best regards

**Comment by progmars** [comment]
It would be at least nice to support more international fonts. Currently here's what I get in Manager: 
![image](https://user-images.githubusercontent.com/4199737/166162260-c5838c8e-6f95-43eb-a820-5d62a96a333c.png)
The browser shows it's using Bitter font and falls back to Times New Roman for the letter ā.

**Comment by GoldSucc** [comment]
> It would be at least nice to support more international fonts. Currently here's what I get in Manager:
> ![image](https://user-images.githubusercontent.com/4199737/166162260-c5838c8e-6f95-43eb-a820-5d62a96a333c.png)
> The browser shows it's using Bitter font and falls back to Times New Roman for the letter ā. I'll have to dig into the docs and source to see how to fix it, since the style seems to be compiled into Piranha.Manager.dll.

It is possible to add styles to manager, this is how you can change a font in editor
